apps/base-docs/base-camp/docs/frontend-setup/wallet-connectors.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wallet Connectors
+description: Learn about how wallet connector libraries aggregate wallets and make it easier to connect to them from your app.
+hide_table_of_contents: false
+---
+
+One of the most intimidating tasks when building an onchain app is making that initial connection between your users' wallets, and your app. Initial research often surfaces a bewildering number of wallets, each with their own SDKs, and own methods to manage the connection. Luckily, you don't actually need to manage all of this on your own. There are a number of libraries specialized in creating a smooth and beautiful user experience to facilitate this connection.
+
+---
+
+## Objectives
+
+By the end of this guide you should be able to:
+
+- Identify the role of a wallet aggregator in an onchain app
+- Debate the pros and cons of using a template
+- Scaffold a new onchain app with RainbowKit
+
+---
+
+## Connecting to the Blockchain
+
+One of the many challenging tasks of building a frontend that can interface with your smart contracts is managing the user's connection between your onchain app and their [EOA] wallet. Not only is there an ever-growing suite of different wallets, but users can (and probably should!) use several different addresses within the same wallet app.
+
+[Rainbowkit] is one of several options that makes this a little bit easier by serving as an aggregator of wallets, and handling some of the details of connecting them. Alternatives include [ConnectKit], and [Dynamic], which are both excellent choices as well.
+
+Each of these include customizable UI/UX components for inviting the user to connect, displaying connection status, and selecting which wallet they wish to use.
+
+### Using the Quick Start
+
+If you're just trying to get up and running as quickly as possible, you can use RainbowKit's [quick start] script to scaffold an app from their template, with a single command. If you're using Yarn:
+
+```bash
+yarn create @rainbow-me/rainbowkit
+```
+
+:::info
+
+The script doesn't accept `.` as a project name, so you'll want to run this script in your `src` directory, or wherever you keep your projects. It will create a folder with the same name as your project, and install the project files inside.
+
+:::
+
+Once it's done, simply run the app with:
+
+```bash
+yarn run dev
+```
+
+Using the script is fast, but it does mean less choice. In this case, it builds the app on top of [Next.js], which is great if you want to use it, but not helpful if you prefer to work from a different framework, such as [Create React App], or [Remix]. The script also doesn't help you if you want to add an onchain integration to an existing site.
+
+---
+
+## Conclusion
```

Comment:
General take, why are we doing Conclusions if we already learn on what we are learning from the Objectives?

I mostly ask becasue when I browse around for docs, I want something easy to read, and streight to the point. Anything too wordy it usually makes me bounce back very fast.

Reply:
The conclusion bookends the learning experience and improves longterm retention by having the reader review and remember the content one last time.

Reply:
I feel like Call To Action at the end are more important than a recap. I personally find recap for a doc that is less than 1k words, not need it.

Recap for books where chapters are like 30 pages, there is need it. As you might have forgot where you start reading.

I understand in theory recap are useful, but I think we disagree in when they should be used. And it's ok we disagree.